explore-analyze/report-and-share/automating-report-generation.md

```diff
@@ -159,3 +165,87 @@ If you experience issues with the deprecated report URLs after you upgrade {{kib
 :::{important}
 In earlier {{kib}} versions, you could use the `&sync` parameter to append to report URLs that held the request open until the document was fully generated. The `&sync` parameter is now unsupported. If you use the `&sync` parameter in Watcher, you must update the parameter.
 :::
+
+## Schedule and share reports [schedule-report-generation]
```

Comment:
Wondering if it would be worth mentioning which report types are compatible with scheduling 🤔 There's only one non-compatible type which is CSV in Lens

Reply:
This would be a good detail to include in the limitations section. I'll add it - thanks!